graphics/bezier_curve.py

```diff
@@ -0,0 +1,107 @@
+# https://en.wikipedia.org/wiki/B%C3%A9zier_curve
+# https://www.tutorialspoint.com/computer_graphics/computer_graphics_curves.htm
+
+from typing import List, Tuple
+import scipy.special as sp
+
+
+class BezierCurve:
+    """
+    A class to generate bezier curves for a given set of points. The implementation works only for 2d points.
+    """
+
+    def __init__(self, list_of_points: List[Tuple]):
+        """
+        The constructor of the Bezier Curve class.
+            list_of_points: List of tuples on which to interpolate.
+        """
+        self.list_of_points = list_of_points
+        self.degree = (
+            len(list_of_points) - 1
+        )  # degree of the curve produced will be n - 1
```

Comment:
Please move the comment to the line above or remove the comment.  For me (who knows nothing about Bezier curves) the comment would tell me _why_ the degree is important or _how_ we will use it.  The code already tells me `n - 1`.